public/app/features/alerting/unified/components/contact-points/utils.ts

```diff
@@ -126,10 +130,27 @@ export function enhanceContactPointsWithMetadata(
     };
   });
 }
+export function isAutoGeneratedPolicy(route: Route) {
+  const simplifiedRoutingToggleEnabled = config.featureToggles.alertingSimplifiedRouting ?? false;
+  if (!simplifiedRoutingToggleEnabled) {
+    return false;
+  }
+  if (!route.object_matchers) {
+    return false;
+  }
+  return (
+    route.object_matchers.some((objectMatcher) => {
+      return (
```

Comment:
Do we not have a utility function for this somewhere already? 🤔

Reply:
I did not find any util doing this 🤷‍♀️